app.json

```diff
@@ -0,0 +1,15 @@
+{
+    "addons": [
+      {
+        "plan": "heroku-postgresql:hobby-dev"
+      }
+    ],
+    "buildpacks": [
+      {
+        "url": "heroku/ruby"
+      }
+    ],
+    "scripts": {
+      "postdeploy": "bundle exec rails db:seed"
```

Comment:
Falla el deploy, dice `Run `rails db:migrate` to update your database then try again.`